src/App.tsx

```diff
@@ -47,6 +47,13 @@ function App() {
       setCheckedCoinBottom(evt);
     }
   };
+
+  const changeToken = (evt: any) => {
+    evt.stopPropagation();
```

Comment:
here we need to make sure that we do not select always the same coin. ie. L-BTC top cannot have L-BTC bottom

Reply:
fixed this.